java/configuration/src/main/java/sleeper/configuration/properties/instance/CompactionProperty.java

```diff
@@ -45,6 +45,16 @@ public interface CompactionProperty {
                     "can be processed again.")
             .defaultValue("0")
             .propertyGroup(InstancePropertyGroup.COMPACTION).build();
+    UserDefinedInstanceProperty COMPACTION_JOB_WAIT_TIME_IN_SECONDS = Index.propertyBuilder("sleeper.compaction.job.wait.time.seconds")
+            .description("The wait time in seconds for compaction jobs to appear on the SQS queue (must be <=20).")
```

Comment:
We can make it clear that this is to do with a compaction task waiting before it terminates. This naming and description suggest that it's to do with the job, when really it's to do with the task.

These descriptions could explain what actually happens when a compaction task waits for jobs to appear and then terminates.

Reply:
Thanks, updated the descriptions to be more specific